bettertest-kafka/src/main/java/org/bitintegro/bettertest/modules/kafka/server/annotation/WithKafkaServer.java

```diff
@@ -0,0 +1,15 @@
+package org.bitintegro.bettertest.modules.kafka.server.annotation;
+
+import org.bitintegro.bettertest.infrastructure.annotation.TestInfrastructureInitializer;
+import org.bitintegro.bettertest.modules.kafka.server.KafkaContainerInitializer;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@TestInfrastructureInitializer(contextInitializer = KafkaContainerInitializer.class)
+public @interface WithKafkaServer {
```

Comment:
Please add JavaDoc.

Please add parameters which allow you to control Kafka version, server configs, etc.